Dockerfile

```diff
@@ -1,13 +1,16 @@
 FROM ubuntu:20.04
```

Comment:
We are building jupyter extension, thus I suggest to use BASE images that have the basic software installed.
Have a look to https://github.com/jupyter/docker-stacks as well as the dockerhub and/or the quay.io repository

Reply:
Changed. Now the base image is [jupyter:minimal-notebook](https://quay.io/repository/jupyter/minimal-notebook). I had to modify a few more lines in the Dockerfile as this image has a conda env with node 12 installed, and the default user does not have enough permissions to execute some of the necessary commands.

Reply:
`minimal-notebook` is fine, but choosing the `:latest` tag is a risky move. We either need to set a specific release, or we can use `quay.io/jupyter/scipy-notebook:python-3.11.8` as an alternative

Reply:
There is also a `:python-3.11.8` tag for `minimal-notebook`, but I changed it to `quay.io/jupyter/scipy-notebook:python-3.11.8`.